### PR TITLE
[WIP] Document SBO Release Notes for GA 1.0

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: enterprise-`4.8`
- **JIRA issues**: [RHDEVDOCS-2926](https://issues.redhat.com/browse/RHDEVDOCS-2926)
- **Preview pages**: [Service Binding Operator release notes](https://deploy-preview-36675--osdocs.netlify.app/openshift-enterprise/latest/applications/application_life_cycle_management/connecting_applications_to_services/sbo-release-notes.html)
- **SME Review**: @pedjak , @baijum 
- **QE review**: @pmacik 
- **Peer-review**:

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@pedjak @baijum @pmacik Please review and provide your feedback. Thanks!

**Comment by Srivaralakshmi** [comment]
Closing this PR due to conflicts. Please refer to the new PR: [Document SBO RN for GA 1.0](https://github.com/openshift/openshift-docs/pull/36691)

File: modules/sbo-release-notes-1-0.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+** Projection of binding data as environment variables.
+* Exposal of binding data from services
+** Based on annotations present in Custom Resource Definitions (CRDs), Custom Resources (CRs), or resources.
+** Based on descriptors present in Operator Lifecycle Manager (OLM) descriptors.
+* Service Binding Options
+** Provision to bind backing services in a namespace that is different from the application namespace.
+** Provision to project binding data into the specific container workloads.
+** Auto-detection of the binding data from resources owned by the backing service CR.
+** Provision to compose custom binding variables from one or more backing services.
+** Custom path projection to provision a custom location to bind the data in your workload that is not compliant with PodSpec.
```
Comment (pedjak) [inline modules/sbo-release-notes-1-0.adoc:52]:
```suggestion
Support for non PodSpec compliant application resources
```

File: modules/sbo-release-notes-1-0.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* Security
+** Role-based access control (RBAC) support to opt-in.
```
Comment (pedjak) [inline modules/sbo-release-notes-1-0.adoc:54]:
```suggestion
Role-based access control (RBAC) support
```

**Pull request closed without merging**